polaris-e2e/cypress/e2e/ui.rename.cy.ts

```diff
@@ -0,0 +1,48 @@
+const {
+  RENAME_DOCUMENT_URN,
+  RENAME_DOCUMENT_CASE_ID,
+  RENAME_DOCUMENT_DOCUMENT_ID,
+} = Cypress.env();
+
+const EXPECTED_DOCUMENT_NAME = "e2e-test-document-name-renamed";
+
+describe("Notes refresh flow", { tags: ["@ci", "@ci-chunk-2"] }, () => {
+  it("can add a note and refresh the tracker", () => {
+    cy.fullLogin();
+    cy.clearCaseTracker(RENAME_DOCUMENT_URN, RENAME_DOCUMENT_CASE_ID);
+
+    cy.visit("/polaris-ui/case-search");
+    cy.setPolarisInstrumentationGuid("PHASE_1");
```

Comment:
For UI tests, this is how we manipulated the correlation id that eventually makes sure we match to the correct mappings as we play through a scenario